### PR TITLE
updating wellbeing checkin components

### LINKED ISSUE #48: Change styling for buttons in Emotion Check in feature so that they match the other buttons on the page
We'll probably need to add material UI to the component.

### LINKED ISSUE #49: Make emoji images larger when hovering over the emoji area to select an emotion.
The selected emoji comes up at a good size but when the user hovers over to view all of the options, the emoji icon choices come up very small

### PR DESCRIPTION
**Pull request by TheObstinatePanda**
# Update to the Well Being Check In card

This was initially in response to [Issue #49](https://github.com/Jamesllllllllll/Daily-Dashboard/issues/49`) to increase the sizing of the emoji options when hovered over. I took the opportunity to style the rest of the well being check in buttons. I had to alter some of the functionality and re-wrote the css files for them to bring them into a standard format.

### Changes
The changes listed below are meant to be comprehensive of meaningful changes that were made. The changes just below the file paths are the changes made to that specific file.

* some CSS files were re-written. The previous versions are also included with ``.old`` included in the file name

```
src/components/LayoutComponents/FeatureCard.js
```
* The style ``overflow: 'visible',`` was added to this element so that the larger components, energy check in and calendar, could scale properly without distorting the parent container.
```
src/components/QuoteOrFact/subComponents/Fact.js
```
* Removed a console.log statement that had previously been commented out
```
src/components/WellBeingCheckin/WellBeingCheckin.js
```
* Added import of styles from new css file ``./WellBeingCheckin.module.css``
* Wrapped ``Stylecard`` content in a div with the styling pointing at ``styles.checkinContainer``
```
src/components/WellBeingCheckin/WellBeingCheckin.module.css
```
* Wrote styles for the ``.checkinContainer`` element
* Implemented flexbox within the container to better organize the buttons
* The width is set so that the buttons do not spread themselves out more than intended
```
src/features/calendar/Calendar.js
```
* Refactored the code so the button does not disappear when the button is clicked, instead it spawns the element above the calendar button
```
src/features/calendar/Calendar.module.css
```
* Rewrote the CSS file to standardize formatting between buttons 
* Added ``align-self: center`` to ``.calendarContainer`` so buttons appear in the center of the containers
* Removed styling for ``.calendarButton`` as they were not applying styles to any of the components
* Added ``.lineChartContainer`` styling to position and style the calendar after it is spawned by a button press
```
src/features/emotionsCheckin/EmotionsCheckin.js
```
* No code changes were made, re-organized the file to make it more readable
```
src/features/emotionsCheckin/EmotionsCheckin.module.css
```
* Rewrote the CSS file to standardize formatting between buttons 
* Moved ``.container`` styling to .emojiPic  to directly work on the main displayed emoji
* Removed styling from ``.emotionsContainer`` (height, width, justify-content, align-content, padding-top)
* Added styling to ``emotionsContainer`` 
  * `padding: .5rem`
  * `align-self: center`
* Removed styling from ``.list`` (flex-direction, margin, justify-content, align-content)
* Added  styling to ``.list``
  * `position: absolute` (needed for z-index)
  * `top: 32%` (positioning from the top)
  * `left: 3.5%` (positioning from the left)
  * `z-index: 2` (displays this element above others on the page)
* Removed ``.emojiPic`` as it was not styling any elements displayed
* Altered``.img`` styling to make sure all emojis have the same width/height.
  * `width: 2rem`
  * `height: 2rem`
```
src/features/energyCheckin/EnergyCheckin.js
```
* Refactored the code so the button does not disappear when the button is clicked, instead it spawns the slider element above the Energy button
```
src/features/energyCheckin/EnergyCheckin.module.css
```
* Rewrote the CSS for this file to simplify the code and standardize the component
* ``.lightning`` now only has the following properties
  * `max-width: 100%`
  * `max-height: 100%`
  * `width: 2rem`
  * `height: 2rem`
* Added ``.energyContainer`` to standardize buttons size
  * `margin: auto`
  * `padding: .5rem`
  * `align-self: center`
* Altered ``.sliderContainer`` so it display is standardized to the component
  * `position: absolute` (needed for z-index)
  * `top: 45%` (positioning from the top)
  * `left: 15%` (positioning from the left)
  * `background-color: #f0f0f0` (filling the background so display matches the buttons)
  * `border: solid black .5px` (frame the overlay so display matches the buttons)
  * `border-radius: 5px;` (softening the corners of the overlay)
  * `padding: .5rem` (give the component some space to breath)
  * `z-index: 2` (displays this element above others on the page)
```
src/features/energyCheckin/energyCheckinSlice.js
```
* refactored the ``toggleSlider`` function to simplify the action. 

### In Conclusion
With these changes, the components within the Well being check in card should function more cleanly within the app. The component's display is rather small so we may think about displaying it differently on the page in future releases.

wellbeing update

### CODE REVIEW

**Comment by eli-cas** [comment]
Nice job Jason! It is looking good!

**Review by eli-cas — APPROVED** [review]
Everything looks good to me!

**Comment by mdwiltfong** [comment]
Do you plan to add tests @TheObstinatePanda ?

**Comment by TheObstinatePanda** [comment]
> Do you plan to add tests @TheObstinatePanda ?

I could add testing components here, possibly. I still need to come back and make some adjustments to the css for viewing the app on mobile.  @eli-cas did the original development of this component, I believe. I'm not sure if she had already planned on adding tests?

Either way, I think that would be a separate pull request.

**Review by eli-cas — APPROVED** [review]
Looks great Jason! Thanks for working on this!
